testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -486,11 +487,7 @@ def choosemeth(sel, meth, periodic):
 
         return sel
 
-    @pytest.mark.parametrize('meth, periodic', [
-        ('kdtree', False),
-        ('distmat', True),
-        ('distmat', False)
-    ])
+    @pytest.mark.parametrize('meth, periodic', methods)
     def test_around(self, u, meth, periodic):
```

Comment:
@jmborr Sorry that I broke the tests again. These tests make sure that  distance based selections done with a KDTree will also work for the periodic case. Unfortunately they currently don't work. This is also broken in the develop branch. It would be nice if you can have a look at that since the main goal of the pkdtree is to work for selections.

Reply:
@kain88-de I'll take a look :)

Reply:
@kain88-de this ain't a quick fix.
All distance-based selections classes in [selection.py](https://github.com/jmborr/mdanalysis/blob/test_pkdtree/package/MDAnalysis/core/selection.py)  have an [`_apply_KDTree`](https://github.com/jmborr/mdanalysis/blob/test_pkdtree/package/MDAnalysis/core/selection.py#L259) method that must be updated to handle the periodic boundary conditions case. 
As of now, the tests are building a non-periodic tree, so whatever results you derive from it will not compare to a call to [distance_array](https://github.com/jmborr/mdanalysis/blob/test_pkdtree/testsuite/MDAnalysisTests/core/test_atomselections.py#L500) which does support periodic boundary conditions.
I would open new issues to implement the periodic tree in the different  distance-based selections. As of now, it should work for non-distance based selections.

Reply:
@kain88-de [AtomNeighborSearch](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/lib/NeighborSearch.py#L38) is used in [test_neighborhood](https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/coordinates/test_pdbqt.py#L58). We could add one additional test here involving periodic conditions. What do you think?